test/server/configuration_impl_test.cc

```diff
@@ -1040,12 +1042,48 @@ TEST_F(ConfigurationImplTest, DEPRECATED_FEATURE_TEST(DeprecatedAccessLogPathWit
   auto bootstrap = Upstream::parseBootstrapFromV3Json(json);
   NiceMock<MockOptions> options;
   NiceMock<Server::MockInstance> server;
-  InitialImpl config(bootstrap, options, server_);
+  InitialImpl config(bootstrap, options);
+  config.initAccessLog(bootstrap, server_);
   Network::MockListenSocket socket_mock;
 
   ASSERT_EQ(config.admin().accessLogs().size(), 2);
 }
 
+TEST_F(ConfigurationImplTest, DEPRECATED_FEATURE_TEST(AccessLogWithFilter)) {
```

Comment:
Not sure why you are using `DEPRECATED_FEATURE_TEST` here; only `access_log_path` field has been deprecated, which you are not using in the test.

Reply:
Could you also add a test to verify that `access_log_path` is handled correctly? You'll need to use `DEPRECATED_FEATURE_TEST` for it.

Reply:
And also a test that verifies that an empty admin config is ok too.